core-web/apps/dotcms-ui/src/stories/primeng/messages/Toast.component.ts

```diff
@@ -1,120 +1,38 @@
-import { Component } from '@angular/core';
+import { Component, Input } from '@angular/core';
 
 import { MessageService } from 'primeng/api';
 
 @Component({
     selector: 'dot-p-toast',
     template: `
         <p-toast></p-toast>
-        <p-toast position="top-left" key="tl"></p-toast>
-        <p-toast position="top-center" key="tc"></p-toast>
-        <p-toast position="bottom-center" key="bc"></p-toast>
+        <p-toast [position]="position" [key]="position"></p-toast>
         <p>
-            <button (click)="showSuccess()" type="button" pButton pRipple label="Success"></button>
-        </p>
-        <p>
-            <button (click)="showError()" type="button" pButton pRipple label="Error"></button>
-        </p>
-        <p>
-            <button (click)="showInfo()" type="button" pButton pRipple label="Info"></button>
-        </p>
-        <p>
-            <button (click)="showWarn()" type="button" pButton pRipple label="Warning"></button>
+            <button
+                (click)="triggerToast()"
+                type="button"
+                pButton
+                pRipple
+                label="Click Me!"></button>
         </p>
     `
 })
 export class ToastComponent {
-    constructor(private messageService: MessageService) {}
-
-    showSuccess() {
-        this.messageService.add({
-            severity: 'success',
-            detail: 'The action "Publish" was executed succesfully'
-        });
-    }
-
-    showInfo() {
-        this.messageService.add({
-            severity: 'info',
-            detail: 'Make sure you add the width of the element'
-        });
-    }
-
-    showWarn() {
-        this.messageService.add({
-            severity: 'warn',
-            detail: 'Make sure you add the width of the element'
-        });
-    }
-
-    showError() {
-        this.messageService.add({
-            severity: 'error',
-            detail: 'Something went wrong, please try again.'
-        });
-    }
+    @Input() severity: string = 'success';
+    @Input() summary: string = 'Success Message';
+    @Input() detail: string = 'The action "Publish" was executed succesfully';
+    @Input() position: string = 'top-right';
+    @Input() life: number = 2000;
```

Comment:
These inputs can be only one input object? Or is toast being used in many places and would complicate the refactor?

Reply:
this is just a component for storybook, we need the inputs separated to pass them using storybook functionality, not worth a refactor